Dockerfile

```diff
@@ -25,6 +25,8 @@ RUN npm install -g /build/saf.tgz
 # Useful for CI pipelines
 RUN apk add bash jq curl ca-certificates
 
+USER ${USER:-node}
```

Comment:
i don't think this will work since we're not passing in a user argument to the dockerfile.  the heimdall dockerfile just sets the user to node and moves on, which i think is fine.  if you wanna provide the user the option to specify which user is actually used (but which defaults to the node user), then you should look into the ARG and ENV dockerfile keywords

Reply:
yeah there's nothing wrong with how you specify the user on this line, but you're not gonna be able to access the $USER variable without passing it in through ARG/ENV.  however, I'm thinking it's fine to just leave it as 'node'.